### PR TITLE
Add smarty blocks everywhere

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add a lot more Smarty blocks in template (see https://github.com/PrestaShop/StarterTheme/pull/167)
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/STAB-65
| How to test?  | You can't, just read the code. Although you can make sure it didnt break any style if you want.

### CODE REVIEW

File: themes/classic/templates/checkout/_partials/cart-detailed.tpl [hunk 1]
```diff
@@ -22,17 +22,19 @@
...
-      {if $product.customizations|count >1}
-      <hr>
-      {/if}
-    {/foreach}
-  </ul>
-  {else}
-    <span class="no-items">{l s='There are no more items in your cart' d='Shop.Theme.Checkout'}</span>
-  {/if}
-</div>
+{block name='cart_detailed_product_line'}
```
Comment (maximebiloe) [inline themes/classic/templates/checkout/_partials/cart-detailed.tpl]: block name already define inside the tpl included line 30

Comment (maximebiloe) [inline themes/classic/templates/checkout/_partials/cart-detailed.tpl]: block name already define inside the tpl included line 30

**Review by maximebiloe — CHANGES_REQUESTED** [review]

**Comment by aleeks** [comment]
Hello @julienbourdeau, you need: 

1. Change
2. Fix for tests
3. Rebase

Hihi, good luck!

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Yeaaah, everything is green.
Seems good to me, thanks @julienbourdeau

**Pull request merged**